must-gather/collection-scripts/gather_nodes

```diff
@@ -62,6 +62,7 @@ do
     oc exec $pod -n perf-node-gather -- lspci -nvv > $NODE_PATH/lspci
     oc exec $pod -n perf-node-gather -- lscpu -e > $NODE_PATH/lscpu
     oc exec $pod -n perf-node-gather -- cat /proc/cmdline > $NODE_PATH/proc_cmdline
+    oc exec $pod -n perf-node-gather -- cat /sys/devices/system/cpu/offline  > $NODE_PATH/offline_cpus
```

Comment:
this works, but we're in a (poorly documented and) deprecated path. Best would be to integrate in `gather_sysinfo`.